src/agent/src/linux_abi.rs

```diff
@@ -15,9 +17,46 @@ pub const SYSFS_PCI_BUS_RESCAN_FILE: &str = "/sys/bus/pci/rescan";
     target_arch = "x86_64",
     target_arch = "x86"
 ))]
-pub const PCI_ROOT_BUS_PATH: &str = "/devices/pci0000:00";
+pub fn create_pci_root_bus_path() -> String {
```

Comment:
can this compile on aarch64? Looks like you are missing cfg macro here otherwise `create_pci_root_bus_path()` is redefined

Reply:
No, the cfg macro above says the function will be compiled on non-aarch64. This is the original logic and I don't change it.